samples/bluetooth/llpm/README.rst

```diff
@@ -0,0 +1,313 @@
+.. _ble_llpm:
+
+Bluetooth: LLPM
+###############
+
+The Bluetooth Low Latency Packet Mode (LLPM) sample uses the :ref:`latency_readme` to showcase the LLPM proprietary Bluetooth extension from Nordic Semiconductor.
+You can use it to determine the transmission latency of LLPM-enabled connections, or to compare with different connection parameters and check their influence on the results.
+
+
+Overview
+********
+
+The LLPM is designed time-sensitive transmissions, in which time synchronization is important and interface response time is critical for the user.
```

Comment:
Would rephrase to: "LLPM is designed for applications where interface response time is critical for the user". Time synchronization is not one of the main use cases